.github/workflows/build.yml

```diff
@@ -300,16 +299,31 @@ jobs:
         run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.46.2
       - name: Run linters
         run: make generate && ./scripts/lint.sh -g lint
-
-      # The go-ethereum and celo-blockchain packages both implement secp256k1 using the exact same header, but that causes duplicate symbols.
-      - name: Run golang tests
-        run: cd node && go test -v -race -ldflags '-extldflags "-Wl,--allow-multiple-definition" ' ./...
       - name: Ensure generated proto matches
         run: |
           rm -rf node/pkg/proto
           docker build --target go-export -f Dockerfile.proto -o type=local,dest=node .
           git diff --name-only --exit-code && echo "✅ Generated proto matches committed proto" || (echo "❌ Generated proto differs from committed proto, run \`rm -rf node/pkg/proto && docker build --target go-export -f Dockerfile.proto -o type=local,dest=node .\` and commit the result" >&2 && exit 1)
 
+  # Run Go tests
+  node-tests:
+    # The tests are slow enough that we want to run it on the self-hosted runner
```

Comment:
are they though? we might be able to get away without using the self-hosted one here

Reply:
Yes, left this comment in the other PR
https://github.com/wormhole-foundation/wormhole/pull/2278#issuecomment-1399630069
> baseline
>
> - self-hosted
>   - lints 15s
>   - tests 5s
> - cloud
>   - lints 4m 50s
>   - tests 12m 50s